packages/openapi-fetch/test/index.test.ts

```diff
@@ -1272,6 +1295,33 @@ describe("client", () => {
         expect(req.headers.get("onFetch")).toBe("exists");
         expect(req.headers.get("onRequest")).toBe("exists");
       });
+
+      it("baseUrl can be overriden", async () => {
```

Comment:
It seems that this test is a duplicate of https://github.com/openapi-ts/openapi-typescript/pull/1817/files#diff-de37e4843cc1c72ff2441d5eed04f070dde7f4ad98e8b2c446bca1243c571d7dR770

The logic is the same for `{ getRequest }` https://github.com/openapi-ts/openapi-typescript/blob/231d5140a56a1922eb738bda6b42efe60d2af1ac/packages/openapi-fetch/test/fixtures/mock-server.ts#L78

Reply:
Are you sure you're talking about the [second test](https://github.com/openapi-ts/openapi-typescript/pull/1817/files#diff-de37e4843cc1c72ff2441d5eed04f070dde7f4ad98e8b2c446bca1243c571d7dR1299) I wrote for the middleware? `{ getRequest }` is not even used there. It's purely a client-side test (doesn't really involve the mock server) which verifies that the overridden baseURL is correctly passed to the middleware and also checks that the trailing slash is removed.

Reply:
As the first test already ensure that the baseUrl is correct on the server (mock server) the second test is kind of a duplicate imo. But as the second one also test that specific parameters (in this case only baseUrl) are passed to the middleware before being used I'm ok to merge this as it is!